sstables/compress.cc

```diff
@@ -157,7 +157,9 @@ void compression::segmented_offsets::state::update_position_trackers(std::size_t
 }
 
 void compression::segmented_offsets::init(uint32_t chunk_size) {
-    SCYLLA_ASSERT(chunk_size != 0);
+    if (chunck_size == 0) {
```

Comment:
typo: `chunck_size`

Reply:
@Jager227 please fix the typo and rebase